05-query-d1/data/csvtosql.py

```diff
@@ -0,0 +1,41 @@
+# script to convert CSV to SQL to upload to Cloudflare D1
+# after running this python script, run on the command line -> npx wrangler d1 execute {your db name} --remote --file=data/data.sql
+import csv
+from pathlib import Path
+
+p = Path(__file__).with_name('quotes.csv')
```

Comment:
Very minor nit: I think it might be a bit clearer to say `Path(__file__).parent / "quotes.csv"`